feature_flag.md

```diff
@@ -4,38 +4,55 @@
 **[this](https://martinfowler.com/articles/feature-toggles.html)**
 **helpful article on the topic.**
 
-## Best practises
+At Companies House we use feature flagging to enable us to deliver features
+without modifying the current behaviours of the system. This may be because
+the features are still being developed or the functionality
+may only become active at a certain date. Other flags may be
+semi-permanent and my be used to toggle functionality on/off as needed (e.g. in
+response to a live support issue.)
+
+The following documentation details how feature flags should be used and
+managed.
+
+## Best practices
 
 ### Naming
 
 * The feature flag name should be descriptive and tell people what the flag
   does before they look for it in the code.
-* It should include a prefix that is both searchable and relevant. The shorter
-  the prefix the harder this will be to search for in our code base. i.e. `FF`
-  is a poor prefix as it is the hexadecimal for 255 a common number in
-  development. Recommendation is to use `FEATURE_FLAG_` which is already used
-  by a number of our code repositories.
+* Feature flags **must** be prefixed with `FEATURE_FLAG_`.
 * The feature flag should contain a creation date in the following format
   `ddmmyy` as a suffix.
 * Avoid using dots in the name where possible as they cause problems on
```

Comment:
Given strengthening of the bullet point above, maybe say here "Do not use dots in the name as they cause..."?